docs/sources/developers/plugins/metadata.md

```diff
@@ -22,25 +22,26 @@ The plugin.json file is required for all plugins. When Grafana starts, it scans
-| `autoEnabled`        | boolean                       | No       | Set to true for app plugins that should be enabled by default in all orgs                                                                                                                                                                                                                                                                                                                               |
+| `autoEnabled`        | boolean                       | No       | Set to true for app plugins that should be enabled and pinned to the navigation bar in all orgs.                                                                                                                                                                                                                                                                                                        |
 | `backend`            | boolean                       | No       | If the plugin has a backend component.                                                                                                                                                                                                                                                                                                                                                                  |
-| `category`           | string                        | No       | Plugin category used on the Add data source page. Possible values are: `tsdb`, `logging`, `cloud`, `tracing`, `sql`, `enterprise`, `other`.                                                                                                                                                                                                                                                             |
-| `enterpriseFeatures` | [object](#enterprisefeatures) | No       | Grafana Enerprise specific features.                                                                                                                                                                                                                                                                                                                                                                    |
+| `builtIn`            | boolean                       | No       | Indicates whether the plugin is developed and shipped as part of Grafana. Also known as a 'core plugin'.                                                                                                                                                                                                                                                                                                |
```

Comment:
This field is a weird one. There's only 3 plugins that set this - `mixed`, `dashboard` and `grafana` datasource plugins - basically "static" datasources that we don't want to show up in the datasources or plugins list. I wonder is it worth documenting this one as it's strictly for internal purposes and ideally we should probably figure out a better way to support these special plugins

Reply:
okay, I'll remove it from the docs and add a note in the schema and plugin def